code_map.py

```diff
@@ -659,10 +678,9 @@ def focus_source_code():
             w.set_view_index(map_view, code_map_group, 0)
             map_view.sel().clear()
 
-            sublime.set_timeout_async(focus_source_code, 10)
-
-            CodeMapListener.active_view = current_view
-            CodeMapListener.active_group = current_group
+            # cannot use CodeMapListener.active_view as it will be immediately
+            # overwritten by other views from the new group being activated
+            sublime.set_timeout_async(lambda: w.focus_view(current_view))
 
```

Comment:
it was true, I wrote it a bit differently because I moved focus_source_code() above